docs/content/test_runner/Metrics.md

```diff
@@ -79,36 +79,52 @@ graph LR
     id1[Test runner start] -->|ms|id2[Test runner finished]
 ```
 
-## `tests.status.lost.`
+## `tests`
+
+Tests running statistics.
+
+### `tests.status.`
+
+Tests runs by statuses.
+
+#### `success`
+
+#### `failure`
+
+#### `skipped`
+
+#### `manual`
+
+#### `lost.`
 
```

Comment:
I'd like to collect information about runs by all statuses to see the whole picture.
The idea is to visualize the proportion of different statuses and see it in a long term.

Described in docs before implementation to discuss options.
@sboishtyan , @dsvoronin , are you ok with these new metrics? Will do?

Reply:
Some of these statuses are in test runner domain, some in report-viewer
Do you really want to mix it?

Reply:
No, I'd like to avoid mixing them. Please help me to distinguish them.

Now we send only `lost` cases. They know about report-viewer, isn't it? E.g. `lost.no-file` from the device.

`TestsRunner` returns `SimpleRunTest` as a result. It is also a model from report-viewer.